schemas/ep00_epics_connection_info.fbs

```diff
@@ -0,0 +1,27 @@
+// Represent events about the underlying EPICS connection.
+
+file_identifier "ep00";
+
+enum EventType: ushort {
+  UNKNOWN,
+  TRY_CONNECT,
+  CONNECT,
+  FAIL,
+  CLOSE,
+  DISCONNECT
+}
+
+struct NanosecondsSinceEpoch {
+  ns: ulong;
+}
+
+table EpicsConnectionInfo {
+  // Nanoseconds since epoch
+  timestamp: NanosecondsSinceEpoch;
```

Comment:
Why not just have a `ulong` field here instead of having the separate struct? That is what we have done in the other schemas.

Reply:
True, but other schemas like `ns00` use seconds (float) instead of nanoseconds. So this here is my attempt at being very explicit about the units being used.

Reply:
Alternatively, what about renaming to `timestamp_ns: ulong;` ?

Reply:
I think that is only the case in the NICOS schemas, all the others the timestamp field is ns. But the `timestamp_ns` suggestion is fine with me.

Reply:
Seeing as we are currently (only) using the `f142` schema to pass EPICS PV values to the filewriter, I think we should try to unify the naming a bit. In this case, instead of `channel_name` use `source_name` and instead of `timestamp_ns` use `timestamp` but properly document that the timestamp is in ns since UNIX epoch.

Reply:
:+1: